code/modules/economy/economy_manager.dm

```diff
@@ -0,0 +1,204 @@
+/*
+/client/proc/library_manager()
```

Comment:
I forgot about this, I'm going to end up deleting this file as I'm saving admin tooling for another PR